### PR TITLE
Better wording for self-signed certificate

### LINKED ISSUE #841: [doc] 4.10.8.2  Self-signed Server Certificate - wording needs to be changed for self signed
4.10.8.2  Self-signed Server Certificate

https://susedoc.github.io/doc-caasp/master/single-html/caasp-admin/#self_signed_certificate
---------------------
This should be changed:

Do not use self-signed certificates in production environments. There is severe risk of Man-in-the-middle attacks. Use proper trusted 3rd-party signed certificates.

To:

Make sure that in case you use self-signed certificates that the Certificate Authority used for signing is configured in a secure way as trusted Certificate Authority on the clients.

Background:

Several customers have their own CA i.e. Active Directory integrated and use this to sign certificates themselves. As they use a secure way (active directory domain join) to trust this CA and the signed certificates - the risk of Man-In-The-Middle attacks is not worth the big red warning.

### PR DESCRIPTION
**Pull request by jenting**
# Make sure it's tested
*Technical writers will not always be able to verify the implementation!*

Any PR opened is assumed to have been verified by QA if not designated by comments or labels otherwise. Please make sure you have tested the changes and included any information that a user would require to use the documentation.

# Describe your changes

Better wording for a self-signed certificate.

# Related Issues / Projects
Please provide links to Bugzilla and other GitHub projects with your description if they are related to the changes.

Fix #841

# Enable maintainer updates
Please enable maintainer updates so we can push commits into your branch to make collaboration and reviews easier.

# Do not force push your branch
Please avoid force pushing to branches that are subject of pull requests. Force pushing breaks maintainer commits in many cases and is very hard (if not impossible) to untangle for backporting.

# Labels
Please set any (and all) appropriate labels that describe the status of the PR.

| **Label** | **Description** |
| --- | --- |
| P1 | PR should be worked on and merged as soon as possible |
| Blocked | Work can not proceed because other work has not been completed, PR can not be merged (code has not been merged but documentation is ready) |
| On-Hold | Underlying work is completed but the PR should not be merged |
| ReleaseNotes | User interaction is required after the introduction of this change and the change must be mentioned in the release notes |
| v3/v4/v4.x | Which version of the release the PR should be merged into, this can be multiple versions, please set the "Backport" label if it needs to go into a previous release |
| Needs Review | Some details of the PR are known to be incomplete and must be discussed with other engineers before merging (if possible assign reviewers or cc mention in comments), PR can not be merged |

Signed-off-by: JenTing Hsiao <jenting.hsiao@suse.com>

### CODE REVIEW

File: adoc/admin-security-certificates.adoc [hunk 1]
```diff
@@ -701,11 +701,10 @@ You should receive the following files in return:
...
 === Self-signed Server Certificate
 
-[WARNING]
+[NOTE]
 ====
-Do not use self-signed certificates in production environments.
-There is severe risk of Man-in-the-middle attacks.
-Use proper trusted 3rd-party signed certificates.
+Make sure that in case you use self-signed certificates that the Certificate Authority
+used for signing is configured in a secure way as trusted Certificate Authority on the clients.
```
Comment (nkoranova) [inline adoc/admin-security-certificates.adoc:707]: This sentence is a bit too long and incoherent. Suggested rewording: 
"In the case that you decide to use self-signed certificates, make sure that the Certificate Authority
used for signing is configured securely as a trusted Certificate Authority on the clients."

Comment (jenting) [reply adoc/admin-security-certificates.adoc]: Updated and rebase onto maintenance/CaaS4 branch

**Comment by jenting** [comment]
@nkoranova Please check the repository branch permission, I could click the Merge pull request button 😉

**Comment by jenting** [comment]
Also, I saw lots of files got changes that are not related to this PR, could you please help check the base branch `maintenance/CaaS4` is up-to-date? thx.

**Comment by jenting** [comment]
Merge remote branch `maintenance/CaaS4`.

**Pull request merged**